recipes/icu/all/conanfile.py

```diff
@@ -32,6 +34,8 @@ class ICUBase(ConanFile):
         "with_unit_tests": False,
         "silent": True,
         "with_dyload": True,
+        "dat_package_file": "None",
```

Comment:
Should it be like this? 🤔 

```suggestion
        "dat_package_file": None,
```